omnibus/config/software/jmxfetch.rb

```diff
@@ -10,13 +10,13 @@
 
 if jmxfetch_version.nil? || jmxfetch_version.empty?
   jmxfetch_version = '0.21.0'
-  jmxfetch_hash = "4feb19275604c275b2c1a3b42f788525bb8bcbb3ec14ef62fe26dca1a58c4f99"
+  jmxfetch_hash = "7ab6c3d53599f3423f50f38c4e427d1166e98785fe6500b959d68e73cfa24491"
```

Comment:
Not ideal that we change the hash for an already "released" jmxfetch, but I guess this is what got uploaded once we automated the build+upload(to bintray) process. Can you confirm this is actually identical sources?

Reply:
Since JAR probably contain some metadata that changes between build we won't get the same hash everytime :(. Does changing to a new version instead of using the same number sounds good to you ?

Reply:
I think you can leave it as is, as this is an internal omnibus definition - unless you want to use a new version number. Like you said, this is almost certainly a metadata issue.